### DISCUSSION TITLE
scaling issues

### DISCUSSION THREAD

**Discussion by salexa2** [TomSchimansky/CustomTkinter#1453] [Q&A]
I have an issue with custom tkinter where, the widgets are scaling to different screen sizes? it scales fine on my laptop but when my teammate tries running the project on his PC and he full screens it (has a way biger monitor = bigger screen size), the frames and widgets look smaller, the only way to change scaling is using the custom widget scaling line but i want the widgets to automatically scale to screen size when in full screen? Is there a way to do this

this is the line I use to create the window size, my friend has a 1950 x 1000 something resolution... could it be the geometry thats dinging us? If so, How would we set the width and height so that the widgets scale automatically to his screen size? I use the .place() method for all my widgets because .grid() had everything looking wonky and .pack() was no better so I'd prefer to keep the .place() train going. Thank you for any help!

root.geometry(f"{1350}x{700}")

![cachedImage](https://user-images.githubusercontent.com/124205990/231638758-2bc57c8f-576b-4830-9c36-82b0fa4c47cf.png)

**Comment by ElectricCandlelight** [comment]
Hard to tell without a minimum example that reproduces the error.

My guess would be it's because your using place. Place can be ok for small apps but I'd avoid using it as it's terrible for a UI that needs to be responsive to window size or screen size changes. Plus it can be harder to maintain, especially if you want to add in more widgets at a later date.

I have an app that has a very similar layout and it uses grid and works great. What do you mean by it looks wonky?

**Comment by salexa2** [comment]
Like when I used .grid to place things, it was still scaling improperly and even had some of the different pages all on the same page (it could possible be because I was using .grid to configure the main page and menu bar . Would I have to change all my instances or .place to grid 😭 or can I just change the .place of the frame that the widgets are on to .grid

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
